cohortextractor/patients.py

```diff
@@ -2858,3 +2858,73 @@ def with_an_ons_cis_record(
             )
     """
     return "with_an_ons_cis_record", locals()
+
+
+
+def with_record_in_UKRR(
+    # picks dataset held by UKRR
+    from_dataset,
+    returning,
+    # Date filtering: date limits
+    on_or_before=None,
+    on_or_after=None,
+    between=None,
+    # Matching rule
+    find_first_match_in_period=None,
+    find_last_match_in_period=None,
+    include_date_of_match=False,
+    date_format=None,
+    return_expectations=None,
+):
+    """
+    Return whether patient has a record in the UK Renal Registry
+
+    Args:
+        from_dataset: string value; options are:
+            * '2019_prevalence' - a prevalence cohort of patients alive and on RRT in December 2019
+            * '2020_prevalence' - a prevalence cohort of patients alive and on RRT in December 2020
+            * '2021_prevalence' - a prevalence cohort of patients alive and on RRT in December 2021
+            * '2020_incidence' - an incidence cohort of patients who started RRT in 2020
+            * '2020_ckd' - a snapshot prevalence cohort of patient with Stage 4 or 5 CKD who were
+                reported to the UKRR to be under renal care in December 2020.
+        returning: string value; options are:
+            * "binary_flag"
+            * "number_of_matches_in_period"
+            * "renal_centre" - string indicating the code of the main renal centre a
+                patient is registered with
+            * "rrt_start_date" - the latest start date for renal replacement therapy
+            * "treatment_modality_start" - the treatment modality at `rrt_start_date` such as
+                ICHD, HHD, HD, PD, Tx
+            * "treatment_modality_prevalence" - the treatment modality from the prevalence data
+            * "latest_creatinine" - most recent creatinine held by UKRR
+            * "latest_egfr" - most recent eGFR held by UKRR
```

Comment:
If it makes more sense, we can map these things to more user-friendly labels (same for the dataset names).  I'm not sure if it's helpful here, but some of the other methods do (e.g. `patients_admitted_to_hospital`).